src/components/MainStage/MainStage.tsx

```diff
@@ -98,11 +99,18 @@ const MainStage = (props: IMainStageProps) => {
 
   const { classes: chatClasses } = useChatStyles()
 
-  if (!mediaStream || !getStreamGuid()) {
+  if (!isAnonymous && (!mediaStream || !getStreamGuid())) {
```

Comment:
will the Anonymous Audience access to the watchparty as well as to the webinar?

Reply:
Presumably? Not sure... good question :)